picasso/src/main/java/com/squareup/picasso/Request.java

```diff
@@ -293,13 +310,43 @@ public Builder clearCenterCrop() {
       return this;
     }
 
+    /** Set the face detector for face center crop. */
+    public Builder setFaceDetector(FaceDetector faceDetector) {
```

Comment:
I guess it is interesting to know that the Android face detector doesnt always produce the same result with the same sources. Sometime it detects a face, sometimes it detects no face on the image.